components/tabs/stories/tabs.stories.js

```diff
@@ -172,39 +167,12 @@ Overflow.parameters = {
 };
 
 /**
- * The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
- */
-export const Sizing = (args, context) => Sizes({
-	Template,
-	withHeading: false,
-	withBorder: false,
-	...args,
-}, context);
-Sizing.args = Default.args;
-Sizing.tags = ["!dev"];
-Sizing.parameters = {
-	chromatic: { disableSnapshot: true },
-};
-
-/**
- * Quiet tabs have no visible divider across the tab items apart from the one that shows the selected tab item. These should be used as sub-level navigation or for containers.
- */
-export const Quiet = QuietGroup.bind({});
-Quiet.args = {
-	isQuiet: true,
-};
-Quiet.tags = ["!dev"];
-Quiet.parameters = {
-	chromatic: { disableSnapshot: true },
-};
-
-/**
- * Compact tabs should never be used without the quiet variation.
+ * In addition to the default, regular density, tabs also come in a compact density which has tighter spacing.
  */
 export const Compact = CompactGroup.bind({});
+Compact.storyName = "Density: Compact";
```

Comment:
Should the "c" in compact be lowercase?

Reply:
Yep! Hmmm, should probably do this in accordion too.